app/helpers/number-format.js

```diff
@@ -1,5 +1,16 @@
 import Ember from 'ember';
+/**
+ * @module ember-preprints
+ * @submodule helpers
+ */
 
+/**
+  * numberFormat helper. Transforms 3500 into 3,500 for example, if in US English locale.
+  *
+  * @class numberFormat.
+  * @param {Integer} element
```

Comment:
In the utterly most pedantic sense, Javascript doesn't have integers; just numbers.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number
http://www.2ality.com/2014/05/is-integer.html

This is mostly only relevant in that YUIDoc can be configured to automatically link to built-in types it knows about. Most people should know what an integer is without looking it up, though.